packages/components/src/dividers/new-messages/new-messages.tsx

```diff
@@ -0,0 +1,31 @@
+import { Stack } from '@tamagui/core'
+import { LinearGradient } from '@tamagui/linear-gradient'
+
+import { Text } from '../../text'
+
+import type { ColorTokens } from '@tamagui/core'
+
+type Props = {
+  color: ColorTokens
+}
+
+const NewMessages = (props: Props) => {
+  const { color = '$blue-50' } = props
+
+  return (
+    <Stack position="relative" height={42} backgroundColor={color}>
+      <LinearGradient
+        height={42}
+        colors={['rgba(255, 255, 255, 0.9)', 'white']}
```

Comment:
Why aren't you using color tokens here? Any special reason?

Reply:
Good catch, thanks. Fixed in https://github.com/status-im/status-web/pull/360/commits/1a0fe57b19c241a90697fe282397c91ff629bf79.